packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -516,17 +526,28 @@ contract StandardValidator is ISemver {
         bytes32 _absolutePrestate,
         uint256 _l2ChainID,
         IProxyAdmin _admin,
+        bool _isSuperGame,
```

Comment:
How does it look if we instead pass in the actual game type here? I feel like that could make the usage a bit clearer since its often hard to remember what a boolean argument means. Alternatively we could use kwargs syntax in the usage of the function.

```
_errors = assertValidPermissionlessDisputeGame(
                _errors, _input.sysCfg, _input.absolutePrestate, _input.l2ChainID, _input.proxyAdmin, false, _overrides
            );
```